sentry/src/routes/channel.rs

```diff
@@ -89,6 +89,7 @@ pub async fn channel_list<A: Adapter>(
         &query.creator,
         &query.validator,
         &query.valid_until_ge,
+        query.page,
```

Comment:
This could be `skip / limit` instead of passing  `query.page` since skip = limit * page.

Reply:
or we remove the skip param and do the calculation in `list_channels`

Reply:
Another option is to return `(total_pages, channels)` since the `db/channel` is handling DB requests and then construct the response here.

```
let (total_pages, channels) = list_channels(...)...;

Ok(ChannelListResponse {
            total_pages,
            total: total_pages,
            page,
            channels,
})
```